tests/components/plugwise/test_config_flow.py

```diff
@@ -172,6 +193,25 @@ async def test_zeroconf_flow(
     assert len(mock_setup_entry.mock_calls) == 1
     assert len(mock_smile_config_flow.connect.mock_calls) == 1
 
+    entries = hass.config_entries.async_entries(config_flow.DOMAIN)
+    assert entries[0].unique_id == "smile12345"
+
+    # Reinitialize flow
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN,
+        context={CONF_SOURCE: SOURCE_ZEROCONF},
+        data=TEST_DISCOVERY,
+    )
+
+    result3 = await hass.config_entries.flow.async_configure(
+        result["flow_id"],
+        user_input={CONF_PASSWORD: TEST_PASSWORD},
+    )
+    await hass.async_block_till_done()
+
+    assert result3.get("type") is FlowResultType.ABORT
+    assert result3.get("reason") == "already_configured"
```

Comment:
This is already tested by `test_zeroconf_abort_anna_with_existing_config_entries` no?

Reply:
On hindsight that test also revolves around trying to configure an Anna when a Adam is already present.

Reply:
@bouwew please confirm, but @joostlek the `test_zeroconf_abort_anna_with_existing_config_entries` tests for configuring Anna while Adam already is present while `test_zeroconf_abort_anna_with_adam` tests aborting configuring an Anna from discovery when an Adam is also discovered

Reply:
If there are more combinations that we should test, let's do that then, as long as it is tested :)

Reply:
There are 3 now at the bottom - the original two for the adam/anna combination as mentioned + the additional one for validating through the mocked configentry as part of the quoted/reviewed code above

Reply:
Added accordingly

Reply:
@CoMPaTech confirm: `test_zeroconf_abort_anna_with_existing_config_entries` tests that discovering the Anna is blocked when the Adam is already present. The other test does what you write :)